ReactFrontEnd/src/build_entries/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import mainLogo from '../../assets/cover1.jpg'
+import styles from './index.css'
```

Comment:
Could be wrong but this looks like an un-used import statement. 
Should just be `import './index.css'`

Reply:
yeah, probably. i whipped it together as a proof of concept in something like 10 minutes. there's no linting standards installed yet either ha ha